django/db/models/sql/compiler.py

```diff
@@ -785,6 +785,9 @@ def as_sql(self, with_limits=True, with_col_aliases=False):
                     ", ".join(sub_selects),
                     " ".join(result),
                 ), tuple(sub_params + params)
+            if self.query.comments:
+                comments = " ".join(f"/*{comment}*/" for comment in self.query.comments)
```

Comment:
Stuck with keeping the comment as verbatim as possible, another alternative would be to add a space between the delimiters and the comment, i.e: 
```suggestion
                comments = " ".join(f"/* {comment} */" for comment in self.query.comments)
```